src/lib/es2015.iterable.d.ts

```diff
@@ -220,259 +220,30 @@ interface String {
     [Symbol.iterator](): IterableIterator<string>;
 }
 
-interface Int8Array {
-    [Symbol.iterator](): IterableIterator<number>;
+interface TypedArray<V extends number | bigint> {
+    [Symbol.iterator](): IterableIterator<V>;
     /**
      * Returns an array of key, value pairs for every entry in the array
      */
-    entries(): IterableIterator<[number, number]>;
+    entries(): IterableIterator<[number, V]>;
     /**
      * Returns an list of keys in the array
      */
     keys(): IterableIterator<number>;
     /**
      * Returns an list of values in the array
      */
-    values(): IterableIterator<number>;
-}
-
-interface Int8ArrayConstructor {
-    new (elements: Iterable<number>): Int8Array;

```

Comment:
When I run tests, I see a lot of failures because these constructors are no longer available. How are they supposed to be made available after this change?

Reply:
Whoops. I'll re-add them.

I've since gotten WSL hooked up, so I *should* be able to run the tests before pushing my update.